WORKSPACE

```diff
@@ -286,3 +288,11 @@ maven_jar(
   name = "org_mockito_mockito_all",
   artifact = "org.mockito:mockito-all:1.10.19",
 )
+
+git_repository(
+  name = "io_bazel_rules_pex",
+  remote = "https://github.com/benley/bazel_rules_pex.git",
```

Comment:
Can these rules get a better home either with the bazel or pex projects? This is fine for testing but we don't want heron to depend on an individual's git project.

Also this will fail in Twitters internal CI build since it's a closed environment without access to github, only a maven repo mirror and a pipy mirror. Could this code be distributed by either of those means somehow?